File: src/Feedback.Analyzer.Backend/Feedback.Analyzer.Domain/Enums/WorkflowStatus.cs
namespace Feedback.Analyzer.Domain.Enums;

/// <summary>
/// Define an enumeration type to represent different statuses of a workflow.
/// </summary>
public enum WorkflowStatus
{
    /// <summary>
    /// Represents that the workflow is queued for execution.
    /// </summary>
    Queued = 0,
    
    /// <summary>
    /// Represents that the workflow is currently running.
    /// </summary>
    Running = 1,
    
    /// <summary>
    /// Represents that the workflow has successfully completed.
    /// </summary>
    Completed = 2,
    
    /// <summary>
    /// Represents that the workflow has failed to complete.
    /// </summary>
    Failed = 3
}

File: src/Insight.Box.FrontEnd/src/modules/workflows/models/WorkflowStatus.ts
/*
 * Defines an enumeration type to represent different statuses of a workflow.
 */
export enum WorkflowStatus {
    /*
     * Represents that the workflow is queued for execution.
     */
    Queued = 0,

    /*
     * Represents that the workflow is currently running.
     */
    Running = 1,

    /*
     * Represents that the workflow has successfully completed.
     */
    Completed = 2,

    /*
     * Represents that the workflow has failed to complete.
     */
    Failed = 3
}